src/main/java/org/kuali/kfs/pdp/batch/service/impl/Iso20022FormatExtractor.java

```diff
@@ -1527,11 +1530,12 @@ private static DocumentType5Code determineDocumentTypeCode(
         } else if (isCreditNote(paymentDetail)) {
             documentTypeCode = DocumentType5Code.CREN;
         } else {
-            LOG.warn(
-                    "determineDocumentTypeCode(...) - PaymentDetail has unexpected DocumentTypeCode : "
-                    + "paymentDetailFinancialDocumentTypeCode={}",
-                    paymentDetail::getFinancialDocumentTypeCode
-            );
+            /*
+             * CU Customization: Allow payments from PDP feeds to be marked as Invoices or Credit Notes.
+             */
+            documentTypeCode = paymentDetail.getNetPaymentAmount().isGreaterEqual(KualiDecimal.ZERO)
```

Comment:
Is the FinancialDocumentTypeCode value not filled when the payment comes from the PDP feed? I was just wondering if there is a way to determine that the payment comes from the PDP feed or is enough to assume that the payment comes from the PDP feed if it is not invoice or credit note? Thank you

Reply:
Given that the FastTrack file generation worked in a similar way I am ok with this assumption. Thank you.